king_grm.R

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(magrittr)
+
+nt_help <- "num.thread argument for snpgdsIBDKING (if NA, detect the number" %>%
+  paste("of cores automatically)")
```

Comment:
why is this string created up here with a `paste` rather than in the `add_argument` call below? I feel like this is an example of adhering to line length limits making the code harder to read rather than easier.

Reply:
Yeah, for line length. I can change it if you feel strongly about it